Makefile

```diff
@@ -12,10 +12,17 @@ pyclean:
 
 test: pyclean
 	pytest tests -rs -v --durations=0
+	pytest tests -rs -v --durations=0 --local
```

Comment:
Since test is just test-remote + test-local, you could rewrite this as `test: test-remote test-local`. This way any change it the way we launch either set of tests is automatically included in the global test target.

Reply:
Nice, thanks !